Mathlib/NumberTheory/RamificationInertia.lean

```diff
@@ -412,11 +408,10 @@ variable (L)
 then the dimension `[S/pS : R/p]` is equal to `[Frac(S) : Frac(R)]`. -/
 theorem finrank_quotient_map [IsDomain S] [IsDedekindDomain R] [Algebra K L]
     [Algebra R L] [IsScalarTower R K L] [IsScalarTower R S L] [IsIntegralClosure S R L]
-    [hp : p.IsMaximal] [IsNoetherian R S] :
+    [hp : p.IsMaximal] [Module.Finite R S] :
```

Comment:
Can you explain the reason behind this change?

Reply:
I find that only `Module.Finite R S` is used in the proof, so I have changed all instances of `IsNoetherian R S` in this file to `Module.Finite R S`. This might be more general and align better with mathematical conventions. Of course, in this theorem, since `R` is a Noetherian ring, `IsNoetherian R S` is equivalent to `Module.Finite R S`.